TNO/Issues/first_test/description.xml

```diff
@@ -0,0 +1,54 @@
+<IssueSpecification version="4">
+	<Name>
+		<en>Demo for TNO</en>
+		<nl>Demo voor TNO</nl>
+	</Name>
+	<ShortName>
+		<en>Demo for TNO</en>
+		<nl>Demo voor TNO</nl>
+	</ShortName>
+	<SchemeManager>irma-demo</SchemeManager>
+	<IssuerID>tnodemo</IssuerID>
+	<CredentialID>demofortno</CredentialID>
+	<Description>
+		<en>A short demo to test functionality, to be expanded later</en>
+		<nl>Een kleine demo om de functionaliteit te testen, om later uitgebreid te worden</nl>
+	</Description>
+	<IssueURL>
+		<en>[TBD]</en>
+		<nl>[TBD]</nl>
+	</IssueURL>
```

Comment:
If you don't want to set a `IssueURL`, so can also completely omit it. Then the app will not trip over a non-parsable URL.